src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InteractiveWindow_InProc.cs

```diff
@@ -213,5 +215,10 @@ public void WaitForReplOutputContains(string outputText)
                 await Task.Delay(50);
             }
         }
+
+        protected override ITextBuffer GetBufferContainingCaret(IWpfTextView view)
```

Comment:
I hope I need the following buffer:

- for Editor 
> view.GetBufferContainingCaret(); 

which is existing

>         public static ITextBuffer GetBufferContainingCaret(this ITextView textView, string contentType = ContentTypeNames.RoslynContentType)

- for Interactive

> _interactiveWindow.TextView.TextBuffer;


Both work for me. Please let me know if you think that the buffer for interactive is defined incorrectly.

I decided to keep GetBufferContainingCaret name as it holds for Editor. It sounds that it also makes sense for Interactive.